### ISSUE TITLE
Qual a ideia Filipe? 

### ISSUE THREAD

**Issue by JeanExtreme002** [filipedeschamps/ideia-unica#2]
Esse é um projeto para um vídeo novo ou já postou o vídeo? Qual a ideia do projeto?

**Comment by bumaruf** [comment]
os cara é ninja bixo KKKKKKKKKK

**Comment by HidekiHrk** [comment]
kkkkkkkkkk a gente sabe que é uma ideia única, só resta esperar

**Comment by lucasromagnoli** [comment]
é @filipedeschamps qual a ideia do projeto?

**Comment by izakdvlpr** [comment]
k

**Comment by filipedeschamps** [comment]
😂 é pro próximo vídeo de segunda 👍

**Comment by Mikael-R** [comment]
Um novo algoritmo que gera ideias únicas 💻

**Comment by jeffersoncbd** [comment]
Deixa eu chutar... este repositório só foi criado para iniciar um novo projeto no gitpod.io e quando o vídeo entrar no ar o @filipedeschamps  vai fazer o push do código gerado pra cá.

**Comment by filipedeschamps** [comment]
😂 vai ser mas simples, é sobre hospedar páginas da forma mais moderna possível 👍

**Comment by aismaniotto** [comment]
É o github pages? ahah eu testei esses tempos com o react, se for, é sensacional

**Comment by gabrielbaldao** [comment]
@filipedeschamps  cria um repositório para galera depositar ideias igual os que você indicou em algum video

**Comment by Niaev** [comment]
Ligeiro

**Comment by JeanExtreme002** [comment]
> @filipedeschamps cria um repositório para galera depositar ideias igual os que você indicou em algum video

Exatamente. Tipo o repositório geekcomputers/Python para a galera criar aplicações sobre quaisquer coisas e criar uma "biblioteca" disso. Seria bem legal :)

**Comment by FelippeChemello** [comment]
Como ele falou que parece um publi, mas não é, provavelmente é a da Vercel, o que realmente ta muito TOP de usar. 
A integração com Next ta muito boa e já tem uma API embutida que facilita para aplicações com baixa complexidade, ou seja, que não precisam de um backend exclusivo. Tenho alguns projetinhos que estou utilizando, exemplo, para consumir uma API externa com autenticação/chave privada, não preciso de um backend pra isso, apenas consumo através da API que o Next disponibiliza.
Ansioso para ver qual vai ser o projeto da vez.

**Comment by aismaniotto** [comment]
@FelippeChemello parece que acertou hehe

**Comment by PeterF521** [comment]
Acho a ideia muito interessante!

**Comment by Wtheodoro** [comment]
É uma ideia deveras massinha aposto. E até vai dar pra abstrair algumas coisas disso. Sério turma

**Comment by PeterF521** [comment]
@filipedeschamps Finalmente vou aprender a colocar o react em produção em nodejs

**Comment by VictorManoel** [comment]
Já estou ansioso pelo vídeo!!!

**Comment by andre2l2** [comment]
Opaaaaa minha chance de contribuir!!!!! Quero saber qual a grande ideia da vez também.

**Comment by 5-FiveSecurity** [comment]
Olá kkm

**Comment by pedroxca** [comment]
issues: serve pra fazer issues pros devs arrumarem
pra que a gente usa:

**Comment by andre-lc-ferreira** [comment]
@pedroxca acho que a gente usa para bater papo kkkkkkkkkkkkkk
